test/cdata-test.js

```diff
@@ -73,6 +73,11 @@ vows.describe('sax_ltx').addBatch({
         ]
         assert.strictEqual(parseChunks(chunks).toString(), '<root>Content</root>')
       }
+    },
+    'Unicode buffers': () => {
+      var buffer = Buffer.from('<root><![CDATA[Ποδήλατο]]></root>', 'utf-8')
+      var el1 = parseChunks([buffer.slice(0, 18), buffer.slice(18)])
+      assert.strictEqual(el1.getText(), 'Ποδήλατο')
```

Comment:
this change is not specific to cdata.

Please move to `unicode-test.js`

Reply:
Didn't notice there was already a unicode-test script. The only issue is that I will have to duplicate the parseChunk code in there, please confirm (alternatively, it could be a secondary method in parse.js).

Reply:
maybe move to `test/helpers.js` ?

not sure if vows will pick that up as a test

Reply:
Tested that, works fine, will update once the rest of the issues are cleared.